docs/docs/development/studio-setup.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Studio Setup
+---
+
+:::warning[Alpha Feature]
+
+ZMK Studio support is in alpha at this point. Although our best efforts are being made to keep compatibility during active development, but is not guaranteed.
+
+:::
+
+This guide will walk you through enabling ZMK Studio support for a keyboard.
+
+The main additional pieces needed for Studio support involve additional metadata needed in order
+to properly to display the physical layouts available for the particular keyboard.
+
+# Physical Layout Positions
+
+Physical layouts are described as part of the [New Shield Guide](./new-shield.mdx#physical-layouts), but additional possible property is not described there. The `keys` property is used to describe the physical attributes of each key position present in that layout, and are listed in the same order as keymap bindings, matrix transforms, etc. The properties available are:
+
+- Width
+- Height
+- X
+- Y
+- Rotation
+- Rotation X
+- Rotation Y
+
+All X/Y/Width/Height values are in "key unit" units, with a 100th of a key unit resolution. Rotations are in degrees, with a 100th of a degree resolution. Due to the nature of devicetree, all values are stored in the devicetree without decimals as hundredths of a value, e.g. one key unit is represented as `100`, twenty degrees is represented as `2000`, etc.
+
+Here is an example physical layout for a 2x2 macropad:
```

Comment:
I think we are missing a paragraph on rotation here, something like:

```suggestion
If the Rotation value is non-zero, the key will be rotated by that angle around the coordinates specified by Rotation X/Y values, after it is translated to X/Y coordinates. Rotation is clockwise for positive values and counter clockwise otherwise.

:::note
You can specify negative values in devicetree using parentheses around it, e.g. `(-3000)` for a 30 degree counter-clockwise rotation.
:::

Here is an example physical layout for a 2x2 macropad:
```

I also think the negative value pointer would be useful here.